pkgs/top-level/python-packages.nix

```diff
@@ -22311,6 +22338,29 @@ in modules // {
     };
   };
 
+  slob = buildPythonPackage rec {
+    name = "slob-20160304";
```

Comment:
Good point. I did this change for both `pythonPackages.slob` and the `xdxf2slob` packages since neither appears to have had proper releases.